tests/test_trainer.py

```diff
@@ -411,5 +412,103 @@ class CurrentTestModel(
     trainer.test()
 
 
+def test_dataloaders_passed_to_fit(tmpdir):
+    """Verify that dataloaders can be passed to fit"""
+    tutils.reset_seed()
+
+    class CurrentTestModel(
+        LightningTestModelBaseWithoutDataloader
+    ):
+        pass
+
+    hparams = tutils.get_hparams()
+
+    # logger file to get meta
+    trainer_options = dict(
+        default_save_path=tmpdir,
+        max_epochs=1,
+        val_percent_check=0.1,
+        train_percent_check=0.2
+    )
+
+    # fit model
```

Comment:
pls split these test as separated tests so it is easier to debug it in future is one of these cases fails...